components/commons/dropshadow.css

```diff
@@ -0,0 +1,38 @@
+/*!
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+%spectrum-DropShadow-emphasized-default {
+  filter: drop-shadow(
+    var(--mod-drop-shadow-emphasized-default-x, var(--spectrum-drop-shadow-emphasized-default-x))
+    var(--mod-drop-shadow-emphasized-default-y, var(--spectrum-drop-shadow-emphasized-default-y))
+    var(--mod-drop-shadow-emphasized-default-blur, var(--spectrum-drop-shadow-emphasized-default-blur))
+    var(--mod-drop-shadow-emphasized-default-color, var(--spectrum-drop-shadow-emphasized-default-color))
```

Comment:
Looks like the color tokens for these aren't defined yet (`emphasized-default-color`, `emphasized-hover-color`, and `elevated-color`), but I'm assuming you're working with design on that already

Reply:
Are you seeing that just in the drop shadow story? I noticed that too, but only for `.spectrum--light`, `.spectrum--dark` shows that the tokens are defined, but they're defined for the light context, which is why I was overriding them.
For light mode:
<img width="1168" alt="image" src="https://github.com/adobe/spectrum-css/assets/54716846/908b1d2f-837a-4645-b8c9-5564f4370b1e">
And dark mode:
<img width="1161" alt="image" src="https://github.com/adobe/spectrum-css/assets/54716846/1f41beb8-a6a2-4db1-a533-565e3dd2d25c">

I'm not really clear on why that's happening, since the tokens appear to have been added in [this PR for 13.0.0-beta.24 and we're above that now](https://github.com/adobe/spectrum-tokens/pull/299), but it seems like it's specific to the styles in Foundations, checking the token for the elevated drop shadow as it's used in color loupe isn't showing the same issue:

For light mode:
<img width="624" alt="image" src="https://github.com/adobe/spectrum-css/assets/54716846/c8b9c8ad-043d-4398-a501-5a80c33dfdd7">
Dark mode:
<img width="628" alt="image" src="https://github.com/adobe/spectrum-css/assets/54716846/d461c462-fb7a-4df6-8500-5621adc7ba8e">